runtime/src/bank.rs

```diff
@@ -1052,7 +1052,7 @@ impl Bank {
             fee_structure: FeeStructure::default(),
             #[cfg(feature = "dev-context-only-utils")]
             hash_overrides: Arc::new(Mutex::new(HashOverrides::default())),
-            accounts_lt_hash: Mutex::new(AccountsLtHash(LtHash([0xBAD1; LtHash::NUM_ELEMENTS]))),
+            accounts_lt_hash: Mutex::new(AccountsLtHash(LtHash::identity())),
```

Comment:
Can you remind me why 0xBAD1 is chosen before?

Reply:
The reason for the change from "BAD1" to identity is that we are not compute and overwrite accounts_lt_hash in this function anymore. Instead, we need to call freeze on the bank to get the hash...